src/main/resources/ca/corefacility/bioinformatics/irida/database/changesets/22.09/rename-ncbi-export-instruments.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="rename-ncbi-export-instruments" author="josh">
+        <update
+                tableName="ncbi_export_biosample">
+            <column name="instrument_model" value="ILLUMINA_MI_SEQ"/>
+            <where>instrument_model='ILLUMINAMISEQ'</where>
+        </update>
+        <update
+                tableName="ncbi_export_biosample">
+            <column name="instrument_model" value="NEXT_SEQ_500"/>
+            <where>instrument_model='ILLUMINANEXTSEQ500'</where>
+        </update>
+        <update
+                tableName="ncbi_export_biosample">
+            <column name="instrument_model" value="AB_SOLID_SYSTEM_2_0"/>
+            <where>instrument_model='ABSOLID2'</where>
+        </update>
```

Comment:
You are missing a migration for `ABSOLID`, please add it.

Reply:
Added in 145ad2eba59c83bd173fc6844b05783ae8294cfa